.travis.yml

```diff
@@ -0,0 +1,56 @@
+stages:
+  - unit_test
+  - integration_test
+
+language: java
+
+services:
+  - docker
+
+jdk:
+  - openjdk11
+
+os:
+  - linux
+
+cache:
+  directories:
+    - ~/.m2/repository
+
+install:
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true --quiet --batch-mode --show-version --activate-profiles travis
+
+.base_unit_test: &base_unit_test
+  stage: unit_test
+  if: type = push
+  script:
+    - mvn -pl "$TEST_SUITE" test
+
+.base_integration_test: &base_integration_test
+  stage: integration_test
+  if: type = pull_request
+  script:
+    - mvn -pl 'beekeeper-integration-tests' -Dtest=$TEST_FILE test
```

Comment:
If we do decide to keep this please use the verbose option for the CLI arguments (`--projects` in this case) as they're more readable.